adapter_pipelines/submit_stub/submit.wdl

```diff
@@ -3,7 +3,7 @@ task submit_stub {
     echo "This is a stub submission task for testing purposes only."
   >>>
   runtime {
-    docker: "python:2.7"
+    docker: "python:2.7-slim"
```

Comment:
Please include a comment here about why we are using the `slim` docker image here